cms/static/js/collections/asset.js

```diff
@@ -37,8 +37,7 @@ define([
             return PagingCollection.prototype.parse.call(this, response, options);
         },
 
-        /* jshint unused:false */
-        parseState: function (response, queryParams, state, options) {
+        parseState: function(response, queryParams, state, options) {  // eslint-disable-line no-unused-vars
```

Comment:
@andy-armstrong Looked into this, it looks safe to just fix this one (I didn't realize the method with extra params wasn't very complex and they're all at the end of the params list)